docs/editor_guide/introduction.rst

```diff
@@ -0,0 +1,23 @@
+Introduction
+============
+
+Wagxperience_ is a brand spanking new open source module for the Wagtail_
+content management system developed by `Lab Digital`_. It allows editors and
```

Comment:
... for Wagtail, Wagexperience is developed by `Lab Digital`

anders wek je de indruk dat wagtail door labd gemaakt is.